### ISSUE TITLE
Failing tests related to media

### ISSUE THREAD

**Issue by supersoju** [WordPress/phpunit-test-runner#93]
We've seen failing tests starting from September 4th on our setup. PHPUnit output is as follows:

1) Tests_Image_Functions::test_pdf_preview_doesnt_overwrite_existing_jpeg
Undefined index: sizes

/var/www/vhosts/wp-test-runner/tests/phpunit/tests/image/functions.php:620

--

There were 3 failures:

1) Tests_Image_Functions::test_wp_generate_attachment_metadata_pdf
Failed asserting that Array &0 () is identical to Array &0 (
    'sizes' => Array &1 (
        'thumbnail' => Array &2 (
            'file' => 'wordpress-gsoc-flyer-pdf-116x150.jpg'
            'width' => 116
            'height' => 150
            'mime-type' => 'image/jpeg'
        )
        'medium' => Array &3 (
            'file' => 'wordpress-gsoc-flyer-pdf-232x300.jpg'
            'width' => 232
            'height' => 300
            'mime-type' => 'image/jpeg'
        )
        'large' => Array &4 (
            'file' => 'wordpress-gsoc-flyer-pdf-791x1024.jpg'
            'width' => 791
            'height' => 1024
            'mime-type' => 'image/jpeg'
        )
        'full' => Array &5 (
            'file' => 'wordpress-gsoc-flyer-pdf.jpg'
            'width' => 1088
            'height' => 1408
            'mime-type' => 'image/jpeg'
        )
    )
).

/var/www/vhosts/wp-test-runner/tests/phpunit/tests/image/functions.php:466

2) Tests_Image_Functions::test_crop_setting_for_pdf
Failed asserting that Array &0 () is identical to Array &0 (
    'sizes' => Array &1 (
        'thumbnail' => Array &2 (
            'file' => 'wordpress-gsoc-flyer-pdf-116x150.jpg'
            'width' => 116
            'height' => 150
            'mime-type' => 'image/jpeg'
        )
        'medium' => Array &3 (
            'file' => 'wordpress-gsoc-flyer-pdf-300x300.jpg'
            'width' => 300
            'height' => 300
            'mime-type' => 'image/jpeg'
        )
        'large' => Array &4 (
            'file' => 'wordpress-gsoc-flyer-pdf-791x1024.jpg'
            'width' => 791
            'height' => 1024
            'mime-type' => 'image/jpeg'
        )
        'full' => Array &5 (
            'file' => 'wordpress-gsoc-flyer-pdf.jpg'
            'width' => 1088
            'height' => 1408
            'mime-type' => 'image/jpeg'
        )
    )
).

/var/www/vhosts/wp-test-runner/tests/phpunit/tests/image/functions.php:531

3) Tests_Image_Functions::test_fallback_intermediate_image_sizes
The `test-size` was not added to the metadata.
Failed asserting that false is true.

/var/www/vhosts/wp-test-runner/tests/phpunit/tests/image/functions.php:572

==================
end test output

Along with this, there was output outside of PHPUnit, which may be related:
`GPL Ghostscript 9.25: Can't find initialization file gs_init.ps.`

**Comment by getsource** [comment]
Hi! Thanks so much for the report!
We talked about this in Slack, and it sounds like it's working again. I'm going to close the issue, but please feel free to reopen if you continue to see the problem.

**Comment by mrxkon** [comment]
Just a follow up here as I was searching like crazy since the same issue was happening on my system and I noticed that PDF was not getting any thumbs at all so the test was correctly failing also.

This isn't related to the tests at  all as it seems.

I managed to fix this by editing the **policy.xml** of my ImageMagick ( i.e. /etc/ImageMagick-6/policy.xml ) and commenting out the line at the very end that says:

`<policy domain="coder" rights="none" pattern="PDF" />`

Tests started passing after this & also PDF generation started working properly within WP media library :) .

Also note to have at least Ghostscript 9.24+ to enable this as it seems there was a vuln report as well 
 https://www.kb.cert.org/vuls/id/332928/
